src/features/slackapi/slackapi-utils.ts

```diff
@@ -47,16 +49,16 @@ namespace SlackApiUtilities {
     } = TimeUtilities.handleTimeConversion(user);
 
     const {
-      underOverMessage,
-      billableHoursWithPercentage
+      message,
+      billableHours
```

Comment:
This is not billableHours but billableHoursPercentage

Reply:
Done